### PR TITLE
#5 Update role for prom2teams v2

### PR DESCRIPTION
**Pull request by jmonterrubio**
Also update CI

### CODE REVIEW

**Comment by jmonterrubio** [comment]
Still need to update the prom2teams and test the logrotate, but you can CR meanwhile

File: templates/logrotate.j2 [hunk 1]
```diff
@@ -0,0 +1,13 @@
+{{ prom2teams_logs_folder }}/uwsgi.log {
```
Comment (jnogol) [inline templates/logrotate.j2:1]: I would use a wildcard, so any log in that folder would be rotated :D

File: molecule.yml [hunk 2]
```diff
@@ -62,7 +66,8 @@ vagrant:
         - prom2teams
       interfaces:
         - network_name: private_network
-          type: dhcp
+          type: static
+          ip: 172.28.128.10
```
Comment (dortegau) [inline molecule.yml:70]: why? :p

File: templates/logrotate.j2 (hunk 1 above)
Comment (jmonterrubio) [reply templates/logrotate.j2:1]: The issue is that there are others that don't need to be rotated by logrotate :)

**Review by jnogol — APPROVED** [review]

File: molecule.yml (hunk 2 above)
Comment (jmonterrubio) [reply molecule.yml:70]: just 4 testing... I could use dhcp if prefered :)

**Review by Gkrlo — APPROVED** [review]

**Review by sorobon — APPROVED** [review]

**Pull request merged**